index.html

```diff
@@ -58,8 +58,10 @@ <h2>Chingu Raptors</h2>
   </section>
   <section id="dino-index">
     <h2>Dinosaur Index</h2>
-    <ul id="dino-list"></ul>
+    <ul id="dino-list"></ul>]
+    <section class="grid-container">
```

Comment:
This container remains empty on the page as it never receives any content, therefore the CSS that corresponds to it is never utilized either.
![image](https://github.com/chingu-voyages/v48-tier1-team-04/assets/33106391/0244e091-20a7-4a69-82c3-0adaa0948afe)

As you can see in the screenshot of our rendered website, the `section.grid-container` is empty.